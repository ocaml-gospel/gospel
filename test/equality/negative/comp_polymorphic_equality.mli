val f : 'a -> 'a -> int
(*@ comparison *)
