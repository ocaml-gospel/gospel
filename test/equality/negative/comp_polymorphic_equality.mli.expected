
*******************************
********** Parsed file ********
*******************************
val f : 'a -> 'a -> int[@@gospel {| comparison |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

val f : 'a -> 'a -> int
(*@ comparison
     *)
File "comp_polymorphic_equality.mli", lines 1-2, characters 0-27:
Error: Type checking error: type is incompatible with equality clause
